.travis.yml

```diff
@@ -15,11 +15,11 @@ install:
   - sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
   # Set up nf-core and test/coverage modules
   - cd ${TRAVIS_BUILD_DIR}
-  - pip install --user -e .
-  - pip install --user codecov
-  - pip install --user nose
+  - sudo pip install .
+  - sudo pip install -r requirements.txt
```

Comment:
Do we need this line? Does `setup.py` not define these already?

Reply:
ah,I am sorry, this is an orphan line I forgot to remove when I tested locally. Had issues with virtualenv and package visibility.